### DISCUSSION TITLE
Is there any documentation supporting the use of Zitadel in cleared environments?

### DISCUSSION THREAD

**Discussion by badrobit** [zitadel/zitadel#8356] [Q&A]
Investigating possible Auth and we need an on-prem capability which zitadel does provide! I was looking to see if there are any documentation or certifications obtained by zitadel to support its use in a US cleared/air-gapped environment?

**Comment by fforootd** [comment]
To my understanding the main concern in these scenarios is the vendor who hosts/operates the services/infra.

Would you mind sharing what certification you are looking for?

**Reply by badrobit** [reply]
not looking for any specific certification we would be looking at the self hosted version so it would more be has the software already been deployed in such an environment (more yes/no without saying anything about which env) or has the gov done a check to determine if the software could be used (they would have reached out to the team maintaining the software).

**Reply by fforootd** [reply]
Got, it. Let me try and answer with that info.

We have customers that use ZITADEL in air-gapped environments in multiple sector (Security, Finance).

Some peopel made request to us about running ZITADEL in Fedram high settings, where we advised them on operational procedures.

**Reply by fforootd** [reply]
To add on this, to some of our customers FIPS compliance is a thing, which we did not yet fully close out.

You can set things like password hashes in zitadel to be compliant but one thing that we not yet addressed is the TLS libraries which are certified. However this is not neccesarly a problem when you run zitadel behind a proxy that is fips compliant.

**Reply by badrobit** [reply]
Thanks that is very helpful. Any chance those operational procedures are available in your documentation? 

FIPS would be a nice to have as I know different parts of the Gov are pushing that harder and harder but we would first have to deploy it to a system with a FIPS compliant kernel such as the [Ubuntu FIPS build](https://ubuntu.com/security/certifications/docs/fips). We could as you say avoid some of the issues by it being behind a FIPS compliant proxy as well. Are you currently pursuing FIPS compliance and if so did you have a timeframe for certification?

**Reply by fforootd** [reply]
We have not ETA for FIPS compliance yet, but if customer demand is high enough we can make it happen.

Checkout #4335 and give it a 👍 

Re, procedures these are not yet publicly documented.
